001-IntroToKubernetes/README.md

```diff
@@ -1,44 +1,46 @@
-   - Deploy MongoDB to AKS from a public container registry.
-- Challenge 7: **[Updates and Rollbacks](Student/07-updaterollback.md)**
-   - Deploy v2 of FabMedical to AKS via rolling updates, roll it back, then deploy it again using the blue/green deployment methodology.
-- Challenge 8: **[Storage](Student/08-storage.md)**
-   - Delete the MongoDB you created earlier and observe what happens when you don't have persistent storage. Fix it!
-- Challenge 9: **[Helm](Student/09-helm.md)**
-   - Install Helm tools, customize a sample Helm package to deploy FabMedical, publish the Helm package to Azure Container Registry and use the Helm package to redeploy FabMedical to AKS.
-- Challenge 10: **[Networking](Student/10-networking.md)**
-   - Explore integrating DNS with Kubernetes services and explore different ways of routing traffic to FabMedical by configuring an Ingress Controller.
-- Challenge 11: **[Operations and Monitoring](Student/11-opsmonitoring.md)**
-   - Explore the logs provided by Kubernetes using the Kubernetes CLI, configure Azure Monitor and build a dashboard that monitors your AKS cluster
-   
+
+- Challenge 00: **[Prerequisites - Ready, Set, GO!](Student/Challenge-00.md)**
+	 - Prepare your workstation to work with Azure, Docker containers, and AKS
+- Challenge 01: **[Got Containers?](Student/Challenge-01.md)**
+	 - Package the "FabMedical" app into a Docker container and run it locally.
+- Challenge 02: **[The Azure Container Registry](Student/Challenge-02.md)**
+	 - Deploy an Azure Container Registry, secure it and publish your container.
+- Challenge 03: **[Introduction To Kubernetes](Student/Challenge-03.md)**
+	 - Install the Kubernetes CLI tool, deploy an AKS cluster in Azure, and verify it is running.
+- Challenge 04: **[Your First Deployment](Student/Challenge-04.md)**
+	 - Pods, Services, Deployments: Getting your YAML on! Deploy the "FabMedical" app to your AKS cluster.
+- Challenge 05: **[Scaling and High Availability](Student/Challenge-05.md)**
+	 - Flex Kubernetes' muscles by scaling pods, and then nodes. Observe how Kubernetes responds to resource limits.
+- Challenge 06: **[Deploy MongoDB to AKS](Student/Challenge-06.md)**
+	 - Deploy MongoDB to AKS from a public container registry.
+- Challenge 07: **[Updates and Rollbacks](Student/Challenge-07.md)**
+	 - Deploy v2 of FabMedical to AKS via rolling updates, roll it back, then deploy it again using the blue/green deployment methodology.
+- Challenge 08: **[Storage](Student/Challenge-08.md)**
+	 - Delete the MongoDB you created earlier and observe what happens when you don't have persistent storage. Fix it!
+- Challenge 09: **[Helm](Student/Challenge-09.md)**
+	 - Install Helm tools, customize a sample Helm package to deploy FabMedical, publish the Helm package to Azure Container Registry and use the Helm package to redeploy FabMedical to AKS.
+- Challenge 10: **[Networking](Student/Challenge-10.md)**
```

Comment:
Need to confirm title as Challenge 10 in the markdown file shows "Networking, DNS, and Ingress"

Reply:
Done.  To make it consistent, I changed it to "Networking and Ingress" everywhere.  We do plan to change/update this challenge in a future PR